src/engine/Actor.ts

```diff
@@ -527,6 +527,48 @@ export class ActorImpl extends Class implements Actionable, Eventable, PointerEv
    */
   public currentDrawing: Drawable = null;
 
+  /**
+   * Draggable helper
+   */
+  private _draggable: boolean = false;
+  private _dragging: boolean = false;
+
+  public get draggable(): boolean {
+    return this._draggable;
+  }
+
+  public set draggable(isDraggable: boolean) {
+    if (isDraggable) {
+      this._draggable = true;
+
+      if (isDraggable) {
+        this.collisionType = CollisionType.PreventCollision;
```

Comment:
Setting the collision type to `PreventCollision` when `draggable = true` may be unexpected for users of Excalibur, I vote we do not set collision type. Thoughts @kamranayub?